src/routes/Details.tsx

```diff
@@ -52,20 +58,64 @@ const Chart = (props: ChartProps): JSX.Element | null => {
       return [djs(time).format(format), close];
     });
 
+    let chartTimePeriod = {
+      MINUTE: "Today",
+      HOUR: "5D",
+      DAY: "1M",
+    }[aggPeriod];
+
+    const lastTrade = parseFloat(lastTradePrice);
+    const firstTrade = parseFloat(aggregates[0].close);
+    const priceDelta = lastTrade - firstTrade;
+    const percentChange = priceDelta / firstTrade;
+
+    const chartColor = percentChange > 0 ? "#059669" : "#dc2626";
+
     return (
       <main>
         <div className="flex flex-b justify-between grow mx-auto w-full pb-2 border-b-2 items-center">
           <div className="flex gap-x-2 justify-between">{name}</div>
+          <div className="items-center flex">
+            <button className="px-4 py-2 inline-flex items-center gap-2 font-semibold text-sm bg-white border-2 border-gray-200 rounded-full shadow-sm">
```

Comment:
Should clicking on this add the current thing to the watchlist?

Reply:
yeah, I was trying to figure that out. needed to see how to maintain states between routes